gradle/dependencies.gradle

```diff
@@ -30,19 +30,19 @@ def apt = [
 ]
 
 def build = [
-    buildToolsVersion: '27.0.3',
-    compileSdkVersion: 27,
+    buildToolsVersion: '28.0.0',
+    compileSdkVersion: 28,
     ci: 'true' == System.getenv('CI'),
     minSdkVersion: 14,
-    targetSdkVersion: 27,
+    targetSdkVersion: 28,
 
     errorProne: "com.google.errorprone:error_prone_core:${versions.errorProne}",
     errorProneCheckApi: "com.google.errorprone:error_prone_check_api:${versions.errorProne}",
     errorProneTestHelpers: "com.google.errorprone:error_prone_test_helpers:${versions.errorProne}",
     nullAway: 'com.uber.nullaway:nullaway:0.3.2',
 
     gradlePlugins: [
-        android: 'com.android.tools.build:gradle:3.1.2',
+        android: 'com.android.tools.build:gradle:3.1.3',
```

Comment:
3.2-beta1?

Reply:
Generally avoid betas since it's a public project